packages/mojatax/src/Utils.js

```diff
@@ -0,0 +1,87 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  assertRelativeUrl,
+  logResponse,
+} from '@openfn/language-common/util';
+import nodepath from 'node:path';
+
+export const authorize = state => {
+  const auth = state.configuration;
```

Comment:
because there's a slim chance that state.configuration is not specified, could we do `const auth = state.configuration ?? {};`